src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -436,9 +436,11 @@ protected virtual StringBuilder BuildBreakInsert(string condition, bool enabled)
                 cmd.Append(" ");
             }
 
+            cmd.Append("\"");
             cmd.Append(filename);
             cmd.Append(":");
             cmd.Append(line.ToString());
+            cmd.Append("\"");
```

Comment:
This is according to the spec for MI. You can't quote the file name alone, you must quote the entire parameter value. So if it doesn't work on lldb it is a bug on their side. However I don't have an apple available to check it. Can one of you do this?  It should be enough to start lldb in MI mode and enter: -break-insert "main.c:10".  But I will leave them off if there are no spaces in the filename and that should prevent any previously working scenario from failing. @pieandcakes